Mail/smtp.php

```diff
@@ -305,7 +305,7 @@ public function send($recipients, $headers, $body)
         $res = $this->_smtp->data($body, $textHeaders);
 		list(,$args) = $this->_smtp->getResponse();
 
-		if (preg_match("/Ok: queued as (.*)/", $args, $queued)) {
```

Comment:
Perhaps you are correct about it being configurable on your server, as everything I can find on this shows the normal server response to be "Ok".